docs/administration.rst

```diff
@@ -30,20 +30,20 @@ Creating Backends
 make ENV=/path/to/environment/config.env createdb
     Create the Data Registry database instance
 
-woudc_data_registry admin setup
+woudc-data-registry admin setup
```

Comment:
Looks like you missed adding the `registry` in here (thanks for catching that underscore though).